src/index.ts

```diff
@@ -130,8 +130,11 @@ async function main(): Promise<void> {
   });
 
   app.get("/", async (_req, res) => {
-    console.log("Login successful.");
-    await session.handleIncomingRedirect(`${iriBase}${_req.url}`);
+    const redirectIri = new URL(_req.url, iriBase).href;
+    console.log(
+      `Login into the Identity Provider successful, receiving request to redirect IRI [${redirectIri}].`
+    );
+    await session.handleIncomingRedirect(`${redirectIri}`);
```

Comment:
I see how this happened, but it's quite a bit too much ceremony for my tastes :P

```suggestion
    await session.handleIncomingRedirect(redirectIri);
```